pkgs/by-name/pi/piglit/package.nix

```diff
@@ -22,21 +29,25 @@
 
 stdenv.mkDerivation {
   pname = "piglit";
-  version = "unstable-2020-10-23";
+  version = "unstable-2025-04-10";
```

Comment:
Doesn’t matter much, but commit d06f7bac988e67db53cbc05dc0b096b00856ab93 was authored on 2025-04-10 and merged on 2025-04-15, so I think 2025-04-15 would be a slightly more accurate date.